py3status/modules/rainbow.py

```diff
@@ -93,11 +98,11 @@ def __init__(self):
     def _init(self):
```

Comment:
Travis is sleeping on the job again. We should fire him.

P.S. I think cycle_time should be renamed to `color_timeout`